tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,587 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "T_eff = 4780  # temperature of hatp11[K], Bakos et al. 2010\n",
+    "\n",
+    "# Used the phoenix module in aesop.py to generate a model spectrum for T_eff = 4780\n",
+    "photlam_hatp11 = SourceSpectrum.from_file('phoenix_model_spectrum.dat', flux_unit = u.erg / u.s / u.cm ** 2 / u.cm)\n",
+    "\n",
+    "# Compute the flux at Earth from the luminosity \n",
+    "R = 0.683 * u.R_sun  # radius in solar units, Deming et al. 2011\n",
+    "d = 123 * u.lyr # distance to HATP11 in light years from Gaia parallax\n",
+    "photlam_hatp11 = photlam_hatp11 * float(R / d) ** 2 / np.pi\n",
+    "\n",
+    "photlam_hatp11.plot(flux_unit='Jy', left=0, right=60000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Simulate the observation with a specific instrument"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Let's observe HAT-P-11 with the Agile instrument on APO's 3.5m telescope, whose net wavelength transmission can be downloaded from <a href=\"https://www.apo.nmsu.edu/arc35m/Instruments/AGILE/Anjum/wav.dat\">APO's website</a>. We will use this to construct a bandpass specific to Agile.</p><p>To construct a bandpass from a file with <tt>synphot</tt>, use <a href=\"https://synphot.readthedocs.io/en/latest/api/synphot.spectrum.SpectralElement.html#synphot.spectrum\">SpectralElement</a>:</p>"
```

Comment:
The Agile exercise was great, but now that we're closing in on reproducing the ARCTIC observations, let's get rid of the Agile example.